scipy/optimize/optimize.py

```diff
@@ -1998,6 +1998,9 @@ def brent(func, args=(), brack=None, tol=1.48e-8, full_output=0, maxiter=500):
     Uses inverse parabolic interpolation when possible to speed up
     convergence of golden section method.
 
+    brent does not ensure that the located minimum is within the initially
```

Comment:
Ah whoops. But how sure are you that this is true? An example proving it would be good.